learn/get-started/create-account.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Create an Account
+---
+
+# Create an Account
+
+The first thing you’ll need to do anything on Stellar is an account. Accounts hold all your money inside Stellar and allow you to send and receive payments—in fact, pretty much everything in Stellar is in some way tied to an account.
+
+Every Stellar account has a pair of public and private keys. Stellar uses public key cryptography in order to ensure that every transaction is secure. The private key is a secret piece of information that proves you own the account, so you should never share it with anyone. It’s kind of like the combination to a lock—anyone who knows it can open the lock. In the same way, anyone who knows your account’s secret key can control your account. The public key is the way other people identify your account and verify that you authorized a transaction.
+
+Because the private key must be kept secret, the first step in creating an account is creating your own keys (when you create the account, you’ll only send the public key to a Stellar server). You can do so with the following command:
+
+<example name="Generating Keys">
+```sh
+# You'll need to install stellar-core for this step.
+$ stellar-core --genseed
+Secret seed: SAV76USXIJOBMEQXPANUOQM6F5LIOTLPDIDVRJBFFE2MDJXG24TAPUU7
+Public: GCFXHS4GXL6BVUCXBWXGTITROWLVYXQKQLF4YH5O5JT3YZXCYPAFBJZB
+```
+
+```js
+// create a completely new and unique pair of keys
+// see more about KeyPair objects: https://stellar.github.io/js-stellar-sdk/Keypair.html
+var pair = StellarSdk.Keypair.random();
+
+pair.seed();
+// SAV76USXIJOBMEQXPANUOQM6F5LIOTLPDIDVRJBFFE2MDJXG24TAPUU7
+pair.accountId();
+// GCFXHS4GXL6BVUCXBWXGTITROWLVYXQKQLF4YH5O5JT3YZXCYPAFBJZB
+```
+
+```java
+// create a completely new and unique pair of keys see more:
+// https://stellar.github.io/java-stellar-sdk/org/stellar/sdk/KeyPair.html
+import org.stellar.sdk.*;
+KeyPair pair = KeyPair.random();
+
+pair.getSecretSeed();
+// SAV76USXIJOBMEQXPANUOQM6F5LIOTLPDIDVRJBFFE2MDJXG24TAPUU7
+pair.getAccountId();
+// GCFXHS4GXL6BVUCXBWXGTITROWLVYXQKQLF4YH5O5JT3YZXCYPAFBJZB
+```
+</example>
+
+[TODO: should this only show if viewing the JS/Java examples?]
```

Comment:
Yeah, that would be really great! I didn’t try and write them because it took me long enough to puzzle out the right approach with the Java SDK (there don’t seem to be any examples anywhere) and I am less familiar with Go to begin with.